gulpfile.js

```diff
@@ -35,11 +35,11 @@ gulp.task("6to5", function () {
 });
 
 gulp.task("sass", function () {
-  return gulp.src("server/src/**/*.scss").pipe(sass()).pipe(gulp.dest("server/dist/css"));
+  return gulp.src("server/src/**/*.scss").pipe(sass()).pipe(gulp.dest("server/dist/static/css"));
```

Comment:
Upon thinking about it further, I think we want a new static/ directory at the top level, next to addon/, server/, and shared/, but we can do that later in another issue